migration-guides.md

```diff
@@ -0,0 +1,131 @@
+# Migration guides
+
+> :star: Switching from other HTTP request libraries to Got :star:
+
+### Migrating from Request
+
+You may think, it's too hard to switch. But that's not true. Let's take the very first example from `request` and see how it looks using `got`.
+
+```js
+const request = require('request');
+request('http://www.google.com', function (error, response, body) {
+  console.log('error:', error); // Print the error if one occurred
+  console.log('statusCode:', response && response.statusCode); // Print the response status code if a response was received
+  console.log('body:', body); // Print the HTML for the Google homepage.
+});
+
+const got = require('got');
+got('http://www.google.com').then(response => {
+	console.log('statusCode:', response.statusCode);
+	console.log('body:', response.body);
+}).catch(error => {
+	console.log('error:', error)
+});
+```
+
+It looks OK, but we can do something about that to make it look better. Let's use `async` functions!
+
+```js
+const got = require('got');
+
+(async () => {
+	try {
+		const response = await got('http://www.google.com');
+		console.log('statusCode:', response.statusCode);
+		console.log('body:', response.body);
+	} catch (error) {
+		console.log('error:', error);
+	}
+})();
+```
+
+Now it looks cool, huh?
+
+#### Options in common
+
+Both `request` and `got` accept [`http.request` options](https://nodejs.org/api/http.html#http_http_request_options_callback).
+Did you know that Got has options which are the same in `request`?
+
+- `url`
```

Comment:
This has different behavior. We support `URL` too.

Reply:
That'll be just a note. The behavior is the same, but we accept additionally `URL`.